### PR TITLE
Expose base64 conversion

### PR DESCRIPTION
**Pull request by tjcorona**

### CODE REVIEW

**Comment by tjcorona** [comment]
@jourdain Would you mind reviewing this merge request?

**Comment by jourdain** [comment]
LGTM but you need to change the commit message otherwise it won't create a release.

`fix(Base64): Expose helper methods to convert base64 to byteArray`

**Comment by jourdain** [comment]
actually expose it as `vtkBase64`

`import vtkBase64 from './Base64';`

**Comment by tjcorona** [comment]
@jourdain I'm not sure why the  CI is failing. What am I doing wrong?

File: Sources/Common/Core/index.js [hunk 1]
```diff
@@ -13,6 +14,7 @@ import vtkURLExtract from './URLExtract';
 import vtkVariantArray from './VariantArray';
 
 export default {
+  Base64,
```
Comment (jourdain) [inline Sources/Common/Core/index.js]: need to be `vtkBase64` now.

Comment (tjcorona) [reply Sources/Common/Core/index.js]: thanks!

**Pull request merged**